lib/index.js

```diff
@@ -118,7 +118,7 @@ module.exports = {
 
 
     const corsMap = {};
-    keys(routes).sort().reverse().forEach((routePath) => {
```

Comment:
api tests did not explode as a result of this being gone... if more ppl wanna test and maybe find a reason why order matters?

Reply:
I don't think it matters, I think the routing engine figures it out correctly